.github/workflows/build.yml

```
@@ -0,0 +1,116 @@
+name: Build
+on:
+  push: { }
+
+env:
+  APP_NAME: ${{ github.event.repository.name }}
+  NAMESPACE: "TrainCrewTIDWindow"
+  TZ: "Asia/Tokyo"
+
+jobs:
+  calc_version:
+    runs-on: ubuntu-latest
+    name: "Calc Version"
+    outputs:
+      VERSION: ${{ steps.calc_version.outputs.VERSION }}
+    steps:
+      - name: Get latest release
+        id: latest_release
+        uses: actions/github-script@v7.0.1
+        if: github.event_name == 'push' && github.ref == 'refs/heads/main'
+        with:
+          script: |
+            try {
+              const response = await github.rest.repos.getLatestRelease({
+                owner: context.repo.owner,
+                repo: context.repo.repo
+              });
+              return response.data.tag_name;
+            }
+            catch (e){
+              if (e.status === 404) {
+                return '';
+              } else {
+                throw e;
+              }
+            }
+      - name: Calculate Version
+        id: calc_version
+        run: |
+          latest_tag="${{ steps.latest_release.outputs.result }}"
+          if [ -n "${latest_tag}" ]; then
+            today=$(date +%Y%m%d)
+            if [[ $latest_tag == $today* ]]; then
+              n=$(echo $latest_tag | awk -F_ '{print $2}')
+              n=$((n + 1))
+            else
+              n=1
+            fi
+            VERSION="${today}_${n}"
+          else
+            VERSION=$(date +%Y%m%d%H%M%S)
+          fi
+          echo "VERSION=$VERSION" >> "$GITHUB_OUTPUT"
+  build:
+    runs-on: ubuntu-latest
+    name: "Build"
+    needs: calc_version
+    strategy:
+      matrix:
+        standalone: [ true, false ]
+        # 接続先をProdにしたバージョンを作成する場合は、リストにfalseを追加する
+        dev: [ true ]
```

現行ではdev環境向けのバージョンのみビルドするが、
以下のように変更を行うことでProd向けのバージョンもビルドすることができる
```suggestion
        dev: [ true, false ]
```